x-pack/platform/plugins/shared/alerting/server/plugin.ts

```diff
@@ -343,6 +345,42 @@ export class AlertingPlugin {
     });
     this.ruleTypeRegistry = ruleTypeRegistry;
 
+    plugins.taskManager.registerTaskDefinitions({
```

Comment:
this task looks like it's just for testing. we should move it to the plugin setup for one of the functional test plugins.

Reply:
Will do